src/components/Modals/Onboarding/OnboardingTransfer.vue

```diff
@@ -327,3 +336,27 @@ export default Vue.extend({
 <style scoped lang="scss">
 @import '@/components/Modals/Onboarding/Onboarding.scss';
 </style>
+<style lang="scss">
+@import '@/components/Modals/Onboarding/Onboarding.scss';
+.new-course {
+  &-onboarding {
+    font-size: 14px;
+    line-height: 17px;
+    color: $black;
+    width: 100%;
+    border-radius: 3px;
+    padding: 0.5rem;
+    border: 0.5px solid $darkPlaceholderGray;
+    border-radius: 0px;
+    background-color: $white;
+    &::placeholder {
+      color: $black;
```

Comment:
Can we make the color `$lightPlaceholderGray` to match how we handle the font color of AP/IB exams that were filled in at a previous time?
![Screen Shot 2021-02-05 at 10 00 21 AM](https://user-images.githubusercontent.com/54298311/107050215-f818d900-6798-11eb-971a-414aa54090ba.png)

Reply:
@tcho6319 color fixed in latest commit